lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -1652,6 +1652,21 @@ void BoundInterpreterFunction::fwdElementSelectInst(
                             I->getLHS()->getElementType(), I);
 }
 
+void BoundInterpreterFunction::fwdModuloInst(glow::ModuloInst const *I) {
+  auto srcH = getTensor(I->getSrc())->getHandle<int64_t>();
```

Comment:
from the doc: Input tensor with int32 or int64 data.
This will fail in debug mode if tensor of int32 data.

Reply:
Good point. Do we have an existing solution for this? Looks like `dispatchFloatingPointImpl`, `dispatchQuantizedImpl`, and `dispatchQuantizedWithAccumulationImpl` don't quite fit this.

Reply:
Yes, Int32ITy and Int64ITy are indexed types, not quantized types. So, you'd need to add something similar.